dask/dataframe/io/parquet.py

```diff
@@ -397,22 +394,23 @@ def _write_pyarrow(df, path, write_index=None, append=False,
     template = fs.sep.join([path, 'part.%i.parquet'])
 
     write = delayed(_write_partition_pyarrow)
-    first_kwargs = kwargs.copy()
-    first_kwargs['metadata_path'] = fs.sep.join([path, '_metadata'])
-    writes = [write(part, open_with, template % i, write_index,
-                    **(kwargs if i else first_kwargs))
+    metadata_path = fs.sep.join([path, '_metadata'])
+    writes = [write(part, open_with, template % i, write_index, compression, metadata_path,
+                    **kwargs)
               for i, part in enumerate(df.to_delayed())]
     return delayed(writes)
 
 
-def _write_partition_pyarrow(df, open_with, filename, write_index,
+def _write_partition_pyarrow(df, open_with, filename, write_index, compression=None,
                              metadata_path=None, **kwargs):
     import pyarrow as pa
     from pyarrow import parquet
     t = pa.Table.from_pandas(df, preserve_index=write_index)
 
     with open_with(filename, 'wb') as fil:
-        parquet.write_table(t, fil, **kwargs)
+        kwargs_copy = kwargs.copy()
+        kwargs_copy['compression'] = compression
+        parquet.write_table(t, fil, **kwargs_copy)
```

Comment:
You can just do `parquet.write_table(t, fil, compression=compression, **kwargs)`